Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -65,20 +70,35 @@ public void setupClimbingCommands(
         
         oi.controller.getXboxButton(XboxButton.Start).whileHeld(aligner);
         
-        oi.operatorPanelButtons.getifAvailable(4).whileHeld(ascend);
+        //oi.operatorPanelButtons.getifAvailable(4).whileHeld(ascend);
```

Comment:
We need to investigate the operator panel - I think one of the switches or buttons is permanently activated. Once we figure that out, we can re-enable most of this commented-out code.